package.json

```diff
@@ -149,7 +149,7 @@
     "webpack-cli": "^4.8.0"
   },
   "scripts": {
-    "start": "DISABLE_WEBPACK=true node dist/backend/index.js",
+    "start": "DISABLE_WEBPACK=true node dist/backend/index.js --unhandled-rejection=warn",
```

Comment:
Is this needed? I think the default should be used, as we have now got handlers for such situations.